modules/GenebassGeneBurden.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python
+"""This module extracts and processes target/disease evidence from the Genebass webservice."""
+
+import argparse
+import logging
+import sys
+
+from pyspark.sql import SparkSession
+from pyspark.sql.dataframe import DataFrame
+from pyspark.sql.functions import coalesce, col, lit, log10, pow, round, when
+from pyspark.sql.types import IntegerType
+
+from common.evidence import initialize_sparksession, read_trait_mappings, write_evidence_strings
+
+METHOD_DESC = {
+    'pLoF': 'Burden test carried out with rare pLOF variants.',
+    'missense|LC': 'Burden test carried out with rare missense variants including low-confidence pLOF and in-frame indels.',
+    'synonymous': 'Burden test carried out with rare synonymous variants.',
+    'pLoF|missense|LC': 'Burden test carried out with pLOF or missense variants.',
+}
+
+
+def main(genebass_data: str, genebass_trait_mappings: str, spark_instance: SparkSession) -> DataFrame:
+    """
+    This module extracts and processes target/disease evidence from the raw Genebass Portal.
+    """
+    logging.info(f'File with the Genebass gene burden results: {genebass_data}')
+    logging.info(f'File with the Genebass traits with their EFO mappings: {genebass_trait_mappings}')
+
+    # Load data
+    genebass_trait_mappings_df = read_trait_mappings(
```

Comment:
[8c3308f](https://github.com/opentargets/evidence_datasource_parsers/pull/130/commits/8c3308f0a5fdd6b13316ef46348ee00ba6f7c650) contains changes about this. I simplified the function and hardcoded the URL of the table, so that the parsers need minimum logic to import it.